src/Tabs/TabsList.js

```diff
@@ -5,7 +5,7 @@ import TabsContext from "./context";
 const noop = () => {};
 
 const TabsList = ({ children }) => {
-  const { tabIds, setTabIds, changeTabs, selectedIndex, hasPanels } =
+  const { tabIds, setTabIds, changeTabs, currentIndex, hasPanels } =
```

Comment:
`currentIndex` is the new internal name for the currently selected index. This prevents naming collisions with the new `selectedIndex` prop